.github/workflows/publish.yml

```diff
@@ -0,0 +1,69 @@
+name: Publish
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: "Version of this release"
+        required: true
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Cache 📦
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Setup Node Environment ⬢
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18
+
+      - name: Tag release
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ github.event.inputs.release_version }}
+          message: "Release ${{ github.event.inputs.release_version }}"
+
+      - name: Update version in package.json
+        id: version-bump
+        uses: jaywcjlove/github-action-package@v1.3.0
+        with:
+          version: ${{ github.event.inputs.release_version }}
+
+      - name: Install 🔧
+        run: npm install
+
+      - name: Conditional
+        id: cond
+        uses: haya14busa/action-cond@v1
+        with:
+          cond: ${{ contains(github.event.inputs.release_version, '-') }}
+          if_true: "next"
+          if_false: "latest"
+
+      - name: Publish 📦
+        id: publish
```

Comment:
Does this do the build before publishing somehow? I think the SDK uses a `prepublishOnly` script, but I don't see that in the package.json for this package.

Reply:
Oh right, I think it was being published manually before. I've added the prepublish step and pointed it at "build:all"